Hashids/Hashids.swift

```diff
@@ -324,8 +323,8 @@ internal func unique<T:CollectionType where T.Generator.Element:Equatable>(a:T)
 
 internal func intersection<T:CollectionType where T.Generator.Element:Equatable>(a:T, b:T) -> [T.Generator.Element]
```

Comment:
I think you should change this method signature rather than adding `b: ` to everyone that calls it. This is a good example of a method where no parameter names are really needed.